cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -0,0 +1,184 @@
+{# ==========================================================================
+
+   Filterable list
+
+   ==========================================================================
+
+   Description:
+
+   Create a filter list that includes controls and results when given:
+
+   value: Collection of field controls
+          and other settings to determine what's rendered.
+
+   value.no_posts_message: Message to show in "no posts" state.
+
+   value.no_posts_explanation: More info to show in "no posts" state.
+
+   value.form_type: What type of form this is.
+
+   value.categories.page_type: Page type of the filterable list control.
+
+   value.output_5050: Whether to render results as info-units or not.
+
+   value.link_image_and_heading: Whether to link the image and heading
+                                 in info-units.
+
+   filter_data.form: Django form that carries the fields to be rendered.
+
+   filter_data.page_set: Result set of posts from the filtered content.
+
+   ========================================================================== #}
+
+{% import 'organisms/filterable-list-controls.html' as filterable_list_controls with context %}
+{% import 'molecules/notification.html' as notification with context %}
+
+{# SHOW A NOTIFICATION IF THERE'S NOTHING TO FILTER. #}
+
+{% if value.no_posts_message %}
+    {{ notification.render(
+        'default',
+        true,
+        value.no_posts_message,
+        value.no_posts_explanation
+    ) }}
+{% elif filter_data.form.filterable_pages.first() %}
+
+    <div class="o-filterable-list">
+        {% set form = filter_data.form %}
+        {% set posts = filter_data.page_set %}
+
+        {# ADD THE FILTER CONTROLS. #}
+
+        {% set fragment_id = 'o-filterable-list-controls' %}
+        {{ filterable_list_controls.render(value, form, fragment_id) }}
+
+        {# ADD THE NOTIFICATION, IF NEEDED. #}
+
+        {% set notification_opts = {
+            'type': 'default',
+            'is_visible': false,
+            'message': '',
+            'explanation': ''
+        } %}
+
+        {% for field in form %}
+            {% if field.errors %}
+                {% for error in field.errors %}
+                    {% if notification_opts.update({
```

Comment:
Why are these `if` statements empty?

Reply:
Andy had the same question. Without having them in an if statement the variable was not visible outside of the scope of the for loop. I added the `if` per the suggestion [here](https://stackoverflow.com/questions/9486393/jinja2-change-the-value-of-a-variable-inside-a-loop). Should I add a code comment?

Reply:
That makes sense 👍. Sure, a comment could be helpful, but isn't 100% necessary.

Reply:
Added a comment above each.